pyuvdata/uvbeam.py

```diff
@@ -294,6 +298,12 @@ def __init__(self):
                                                        form='str', expected_type=str,
                                                        description=desc,
                                                        acceptable_vals=list(self.interpolation_function_dict.keys()))
+        desc = ('String indicating frequency interpolation kind. '
+                'See scipy.interpolate.interp1d for details. Default is linear.')
+        self._freq_interpolation_kind = uvp.UVParameter("freq_interpolation_kind",
+                                                        required=False, form='str',
+                                                        expected_type=str, description=desc)
+        self.freq_interpolation_kind = 'linear'
```

Comment:
This is great, thank you.

I think we need essentially the same thing for the frequency interpolation tolerance.